src/common/module.c

```diff
@@ -15,6 +15,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#if defined(__FreeBSD__)
+#include <sys/wait.h>
```

Comment:
@joscollin 
I'm not quite sure why these would be newly added.
The older version I have does have this include.
It even works without the `#if`